### PR TITLE
[READY] Ronald/meta

### PR DESCRIPTION
**Pull request by RonaldDijkstra**

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -53,4 +53,22 @@
...
+      when "en"
+        "en_US"
+      else
+        "#{lang.downcase}_#{lang.upcase}"
+    end
+  end
+
+  # Get full url
+  def full_url(url)
+    URI.join("http://www.learningspaces.io", url)
```
Comment (fatboypunk) [inline config.rb]: So everything will always be `.io`?

File: config.rb [hunk 2]
```diff
@@ -53,4 +53,22 @@
...
     Tilt['markdown'].new { string }.render(scope=self)
   end
+
+  def full_locale(lang=I18n.locale.to_s)
+    case lang
+      when "en"
+        "en_US"
+      else
+        "#{lang.downcase}_#{lang.upcase}"
+    end
```
Comment (fatboypunk) [inline config.rb]: instead of a case, we could use an `if` `else` statement here

File: source/layouts/layout.erb [hunk 3]
```diff
@@ -3,11 +3,13 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="x-ua-compatible" content="ie=edge">
-    <title><%= current_page.data.title || "LearningSpaces" %></title>
-    <meta name="description" content="">
+    <title><%= current_page.data.title || "LearningSpaces" %><%= " | LearningSpaces" unless current_page.path == "index.html" %></title>
```
Comment (fatboypunk) [inline source/layouts/layout.erb]: so if there is a page without a title it'll be `LearningSpaces | LearningSpaces` ?

File: config.rb (hunk 1 above)
Comment (RonaldDijkstra) [inline config.rb]: @sn3p can we make this localizable?

**Comment by sn3p** [comment]
Also this is still based on Middleman v3 and and some things are broken in v4.
Will fix and merge tomorrow.

File: source/layouts/layout.erb (hunk 3 above)
Comment (fatboypunk) [inline source/layouts/layout.erb]: As a reference: I spoke with Ronald about this and the best thing for this seems to be to use the [middleman-title gem](https://github.com/jcypret/middleman-title)

**Pull request merged**